res/database/Default/database_news.xml

```diff
@@ -5736,6 +5737,33 @@ W tym roku ponownie spodziewane są demonstracje.</pl>
 			<data genre="2" price="1" quality="30" fictional="1" />
 		</news>
 
+		<news id="db08bc32-13b2-45b9-939e-89f5debafdcc" type="0" thread_id="e4e9ad4b-e43e-442c-aba9-971f92441b3a" creator="" created_by="-" comment="TODO: variations">
+			<title>
+				<de>BTX-Poker immer beliebter</de>
+				<en>BBS poker increasingly popular</en>
+			</title>
+			<description>
+				<de>Marktforscher stellen fest, dass unter den elektronischen Mailbox-Spielen besonders das Pokern beliebt geworden ist. Es wird oft um hohe Preisgelder gespielt.</de>
+				<en>Market researchers have found that poker has become particularly popular among electronic bulletin board games. It is often played for high prize money.</en>
+			</description>
+			<data genre="3" price="1" quality="25" fictional="1" />
```

Comment:
Einmalevent?

Reply:
not for now, eventually there'll be a big news overhaul, there are a lot of entries that should be one-time or have variations, removing all them will result in not much being left I'm afraid.